### ISSUE TITLE
Consider increasing time until expiration for job results

### ISSUE THREAD

**Issue by naglepuff** [microbiomedata/nmdc-server#821]
Our task results are stored in redis, and results only stick around for 1 day. This can be frustrating for a job that can now take 48 hours +. This can be [configured in our celery app](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-expires). Increasing the amount of time we hold on to job results (e.g. for a week or forever) could be helpful for investigating jobs.

For example, if an ingest task fails, the details about that failure only persist for 1 day. If the error is not caught before then, it can be difficult to debug without retrying and checking the redis store every so often. If we hang on to the result longer, there is less of a chance we lose information in this manner.

**Comment by jeffbaumes** [comment]
Makes sense. @naglepuff if it's as easy as changing a setting, I suggest you go ahead and make the change.
